codespell_lib/_codespell.py

```diff
@@ -642,6 +644,15 @@ def parse_file(filename, colors, summary, misspellings, exclude_lines,
         if line in exclude_lines:
             continue
 
+        extra_words_to_ignore = set()
```

Comment:
Maybe a more useful variable name like local_words_to_ignore or ignore_words_line or something.

Reply:
I don't like your suggestions.  Postpone.  What it holds is "additional (extra) words to ignore on this line".

Reply:
Perhaps `words_to_ignore_in_line`?